<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectManager</class>
 <widget class="QDialog" name="ProjectManager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreateNew  Project</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="finish">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Project Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="7" colspan="2">
    <widget class="QTableView" name="tableView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="removeButton">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="addFileButton">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Import Table</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="addImageAsRGB">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Import Images</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>180</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="addImageListButton">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Add Obj/Var Depiction </string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Import Text</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
